test/test.g.dart

```diff
@@ -18,7 +18,7 @@ class Car extends _Car with RealmEntity, RealmObject {
   @override
   String get make => RealmObject.get<String>(this, 'make') as String;
   @override
-  set make(String value) => throw RealmUnsupportedSetError();
+  set make(String value) => RealmObject.set(this, 'make', value);
```

Comment:
It will be nice to re-generate the models also in dart and flutter examples under realm-dart folder.